spec/services/email_timetable_spec.rb

```diff
@@ -78,4 +78,18 @@
       end
     end
   end
+
+  describe '#send_reject_by_default_explainer_to_candidates?' do
+    it 'returns false if it is not the day after the reject by default date' do
+      travel_temporarily_to(described_class.reject_by_default) do
+        expect(described_class.send_reject_by_default_explainer_to_candidates?).to be false
+      end
+    end
+
+    it 'returns true when it is the day after the reject by default date' do
+      travel_temporarily_to(described_class.reject_by_default + 1.day) do
```

Comment:
Do you think it's worth testing that it returns `false` if it's `+ 1.hour` and ` + 2.days`

Reply:
I'm not sure it really adds value. it's either on the date, before or after the date. Like if you think of it on a timeline. There is no difference between 1.hours and 2.days.

Reply:
I think it's ok, I was probably over thinking it with potential edge cases with that 👇🏼 

current_date.to_date == (reject_by_default + 1.day).to_date